flink-clients/src/test/java/org/apache/flink/client/program/rest/RestClusterClientTest.java

```diff
@@ -542,6 +549,68 @@ public void testListJobs() throws Exception {
 		}
 	}
 
+	@Test
+	public void testGetAccumulators() throws Exception {
+		TestAccumulatorHandlers accumulatorHandlers = new TestAccumulatorHandlers();
+		TestAccumulatorHandlers.TestAccumulatorHandler accumulatorHandler = accumulatorHandlers.new TestAccumulatorHandler();
+
+		try (TestRestServerEndpoint ignored = createRestServerEndpoint(accumulatorHandler)){
+
+			JobID id = new JobID();
+
+			{
+				Map<String, Object> accumulators = restClusterClient.getAccumulators(id);
+				assertNotNull(accumulators);
+				assertEquals(1, accumulators.size());
+
+				assertEquals(true, accumulators.containsKey("testKey"));
+				assertEquals("testValue", accumulators.get("testKey").toString());
+			}
+		}
+	}
+
+	private class TestAccumulatorHandlers  {
```

Comment:
Why having this extra wrapping class?

Reply:
this follows the `TestSavepointHandlers` in `RestClusterClientTest`, I will remove it.